manim/mobject/coordinate_systems.py

```
@@ -495,3 +503,279 @@ def get_coordinate_labels(self, *numbers, **kwargs):
+            polarplot_gradians = PolarPlane(azimuth_units="gradians").scale(0.4).add_coordinates()
+
+            class PolarPlaneUnits(Scene):
+                def construct(self):
+                    self.add(polarplot_pi)
+                    self.wait(2)
+                    self.play(FadeOutAndShift(polarplot_pi, LEFT))
+                    self.play(FadeInFrom(polarplot_tau, RIGHT))
+                    self.wait(2)
+                    self.play(FadeOutAndShift(polarplot_tau, LEFT))
+                    self.play(FadeInFrom(polarplot_degrees, RIGHT))
+                    self.wait(2)
+                    self.play(FadeOutAndShift(polarplot_degrees, LEFT))
+                    self.play(FadeInFrom(polarplot_gradians, RIGHT))
+                    self.wait(2)
+
+
+    azimuth_offset : :class:`float`, optional
+        The angle offset of the azimuth labels, expressed in radians.
+
+    azimuth_direction : :class:`str`, optional
+        The direction of the azimuth labels.
+
+            - ``"CW"``: Clockwise.
+            - ``"CCW"``: Anti-clockwise.
+
+    azimuth_buff : :class:`int`, optional
+        The buffer for the azimuth labels.
+
+    radius_config: :class:`dict`, optional
+        The axis config for the radius.
+
+    radius_max : :class:`float`, optional
+        The maximum value of the radius.
+    """
+
+    def __init__(
+        self,
+        color=BLUE,
+        azimuth_line_frequency=None,
+        radius_line_frequency=1,
+        azimuth_units="PI radians",
+        azimuth_offset=0,
+        azimuth_direction="CCW",
+        azimuth_buff=SMALL_BUFF,
+        radius_config={"label_direction": DOWN + LEFT},
+        radius_max=None,
+        **kwargs,
+    ):
+        if azimuth_units in [
+            "PI radians",
+            "TAU radians",
+            "degrees",
+            "gradians",
+            None,
+        ]:
+            self.azimuth_units = azimuth_units
+        else:
+            ValueError(
```

If you want to throw an error, you have to `raise` it.

```suggestion
            raise ValueError(
```